WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -597,8 +597,7 @@ private extension PostListViewController {
     func handleRefreshNoResultsViewController(_ noResultsViewController: NoResultsViewController) {
 
         guard connectionAvailable() else {
-            noResultsViewController.configure(title: noConnectionMessage(),
-                                              image: noResultsImageName)
+            noResultsViewController.configure(title: "", noConnectionTitle: NoResultsText.noConnectionTitle, buttonTitle: NoResultsText.noConnectionButtonTitle, subtitle: nil, noConnectionSubtitle: NoResultsText.noConnectionSubtitle, attributedSubtitle: nil, attributedSubtitleConfiguration: nil, image: nil, subtitleImage: nil, accessoryView: nil)
```

Comment:
Considering that `NoResultsViewController` is prepared to handle the offline state, why not avoid this guard here entirely and send the offline customizations to the calls below?

Reply:
As discussed let's skip this for now as understanding the existing code ain't simple.